packages/bff/src/routers/catalogRouter.ts

```diff
@@ -60,7 +62,25 @@ const catalogRouter = (
     )
     .get(
       "/producers/eservices/:eserviceId/descriptors/:descriptorId",
-      async (_req, res) => res.status(501).send()
+      async (req, res) => {
+        const ctx = fromBffAppContext(req.ctx, req.headers);
+
+        try {
+          const response = await catalogService.getProducerEServiceDescriptor(
+            unsafeBrandId(req.params.eserviceId),
+            unsafeBrandId(req.params.descriptorId),
+            ctx
+          );
+          return res.status(200).json(response).send();
+        } catch (error) {
+          const errorRes = makeApiProblem(
+            error,
+            bffGetCatalogErrorMapper,
+            ctx.logger
+          );
```

Comment:
```suggestion
            ctx.logger,
            `Error retrieving producer descriptor ${req.params.descriptorId} for eservice ${req.params.eserviceId}`
          );
```
Since #745 is merged we should include this kind of logging on routes as in Scala

Reply:
Added in [this commit](https://github.com/pagopa/interop-be-monorepo/pull/712/commits/11026a33d1f115e951999825e541b7d14bddba04)